src/main/java/com/brightcove/castlabs/client/CastlabsClient.java

```diff
@@ -222,38 +213,75 @@ public AddSubMerchantAccountResponse addSubMerchantAccount(final AddSubMerchantA
-            final RequestConfig requestConfig =
-                    RequestConfig.custom().setConnectionRequestTimeout(connectionTimeout)
-                            .setConnectTimeout(connectionTimeout)
-                            .setSocketTimeout(connectionTimeout).build();
-            addResellerRequest.setConfig(requestConfig);
-        }
-        addResellerRequest.setEntity(new StringEntity(objectMapper.writeValueAsString(request)));
+        final HttpPost httpRequest = createHttpPostRequest(uri, request);
 
         final CloseableHttpClient httpclient = HttpClients.createDefault();
-        try (final CloseableHttpResponse httpResponse = httpclient.execute(addResellerRequest)){
+        try (final CloseableHttpResponse httpResponse = httpclient.execute(httpRequest)) {
             final HttpEntity responseEntity = httpResponse.getEntity();
             if (responseEntity == null) {
                 throw new CastlabsException("Empty response entity from Castlabs. HTTP Status: " + httpResponse.getStatusLine().getStatusCode());
             }
 
             final String responseBody = IOUtils.toString(responseEntity.getContent());
-            if(StringUtils.isBlank(responseBody)) {
+            if (StringUtils.isBlank(responseBody)) {
                 throw new CastlabsException("Empty response entity from Castlabs. HTTP Status: " + httpResponse.getStatusLine().getStatusCode());
             }
 
             final AddSubMerchantAccountResponse response = objectMapper.readValue(responseBody, AddSubMerchantAccountResponse.class);
-            if(response.getSubMerchantUuid() == null) {
+            if (response.getSubMerchantUuid() == null) {
                 throw new CastlabsException("Unexpected response from Castlabs: " + responseBody);
             }
             return response;
         }
     }
 
+    /**
+     * Add a sub merchant account to Castlabs.
+     *
+     * @param request      Request parameters to pass to Castlabs
+     * @param resellerUuid UUID for the merchant that the sub-merchant was created off
+     * @throws CastlabsException error reported by Castlabs
+     * @throws IOException       network error while communicating with Castlabs REST API
+     */
+    public void linkAccountToSubMerchant(final LinkAccountToSubMerchantRequest request, final String resellerUuid)
+            throws IOException, CastlabsException {
+
+        final String uri = this.getUrlWithTicket(this.ingestionBaseUrl + "frontend/rest/reselling/v1/reseller/" + resellerUuid + "/submerchant/linkAccount");
+        final HttpPost httpRequest = createHttpPostRequest(uri, request);
+
+        final CloseableHttpClient httpclient = HttpClients.createDefault();
+        try (final CloseableHttpResponse httpResponse = httpclient.execute(httpRequest)) {
+            final int statusCode = httpResponse.getStatusLine().getStatusCode();
+
+            if (statusCode < 200 || statusCode >= 300) {
```

Comment:
I'd be surprised if `org.apache.http` didn't have constants for these somewhere. Could we double-check?